scripts/docStories.js

```diff
@@ -0,0 +1,55 @@
+
+/**
+ * This script runs and generates the corresponding documentation {Component}.stories.mdx files for each package if it doesn't already exist.
+ */
+
+const fs = require("fs");
+const shell = require("shelljs");
+
+const skipPackages = [
+  "BuildTranslations",
+  "Calendar",
+  "Constants",
+  "helpers",
+  "Stylers",
+  "Overlay",
+];
+
+const renderMDXFileTemplate = ({
+  displayName = "",
+}) => {
+  return `
+import { Meta, Description } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import { docsStoryParameters } from "storybook/assets/storyParameters";
+import Readme from "../README.md";
```

Comment:
shouldn't we check if the story has a readme file? in case of lacking of one this import will crash the entire mdx file.

Reply:
good point, we should have that!